### PR TITLE
TAPP-52: Fixing "No columns found" overlap issue

### PR DESCRIPTION
**Pull request by dingtaoliu**
updating ReactTable css so that the table is fully expanded to prevent the noDataText from hovering over the headers

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 tmp
+.idea
```
Comment (chrislessard) [inline .gitignore]: I'm ok with this addition, but could you please move it into its own commit so that changes to the file dont get lost in the minutia of other issues and work ?

File: frontend/src/styles/sass/components/_react-table.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/**
+ * A hack to make sure that the rt-noData element does not overlap the table headers
+ */
+
+.positions-table {
+  height: 100vh
+}
```
Comment (chrislessard) [inline frontend/src/styles/sass/components/_react-table.scss]: Please add a newline to the end of the file

File: frontend/src/styles/sass/components/_module.scss [hunk 3]
```diff
@@ -11,3 +11,4 @@
 @import "_load.scss";
 @import "_modal.scss";
 @import "_table-container.scss";
+@import "_react-table.scss";
```
Comment (chrislessard) [inline frontend/src/styles/sass/components/_module.scss]: Please add a newline to the end of the file

**Review by chrislessard — CHANGES_REQUESTED** [review]
The fix looks good to me, there are just a few minor changes to apply !

**Review by chrislessard — CHANGES_REQUESTED** [review]
Could you also please reformat the commit style to follow https://github.com/uoft-tapp/tapp/issues/39 ?

**Comment by dingtaoliu** [comment]
I've made changes according to the comments, please take a look and merge if it is okay!

**Comment by chrislessard** [comment]
@dingtaoliu Your commit messages still aren't following the standard we agreed on in #39. Could you please change them ? Otherwise, the code is all good :)

**Review by chrislessard — APPROVED** [review]
